proto/internalapi/central/compliance_operator.proto

```diff
@@ -266,3 +266,13 @@ message ComplianceOperatorSuiteV2 {
   reserved 3;
   ComplianceOperatorStatus status = 4;
 }
+
+message ComplianceOperatorRemediationV2 {
+  string id = 1;
+  string name = 2;
+  string compliance_check_result_name = 3;
+  bool apply = 4;
+  string current = 5;
+  string outdated = 6;
```

Comment:
Do these come from CO?  What do these fields mean?  Wondering if we could provide a little more context in the name.

Reply:
Yes, this is the same name as in CO. I've named them 1:1 like the CR fields. If it is not a pattern, I also in favor of changing it to a more descriptive name, but keep it clear that it maps to a CR value.

Reply:
Up to you.  They don't have to match particularly in the storage and external API side.   Comment of what they mean at a minimum would be useful.